libs/login/WordPressLoginFlow/src/main/res/values/strings.xml

```diff
@@ -22,7 +22,7 @@
     <string name="open_mail">Open mail</string>
     <string name="alternatively">Alternatively:</string>
     <string name="enter_site_address_instead">Log in by entering your site address.</string>
-    <string name="enter_site_credentials_instead">Log in with site credentials</string>
+    <string name="enter_site_credentials_instead">Log in with site credentials.</string>
```

Comment:
This change should be replicated [here](https://github.com/wordpress-mobile/WordPress-Android/blob/16fdfdad8ded1d2ae9cff3c9d97ed2cc2066ca20/WordPress/src/main/res/values/strings.xml#L2414) for the sake of linting.

Reply:
Fixed in 66ea2cd38c5e57f33d1ed55579063e15b508b8f0 👍